contact.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+
+<head>
+    <title>GameHub</title>
```

Comment:
The title must be unique per page. For example, the title of this page could be `contact`